python/py-keep/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-keep
+version             2.10.1
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         A Meta CLI toolkit
+long_description    A Meta CLI toolkit: your personal shell command keeper
+homepage            https://pypi.org/project/keep/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            keep-${version}
+checksums           rmd160  5bef8733deb9231da85b96993d381eb169533909 \
+                    sha256  3abbe445347711cecd9cbb80dab4a0777418972fc14a14e9387d0d2ae4b6adb7 \
+                    size    13044
+
+python.versions     37 38 39 310
+
+if {${name} ne ${subport}} {
+   depends_build-append port:py${python.version}-setuptools
+   depends_lib-append   port:py${python.version}-terminaltables \
```

Comment:
these two lines are not aligned properly with four spaces

Reply:
Should be fixed now.